repo/plugin.video.nzbdav/resources/lib/player_installer.py

```
@@ -102,6 +102,66 @@ def discover_other_player_targets():
+            "NZB-DAV: Player already installed at schema v{}; "
+            "preserving existing file".format(_PLAYER_SCHEMA_VERSION),
+            xbmc.LOGINFO,
+        )
+        _notify(_addon_name(), _fmt(30094, target_name))
+        return True
+
+    # Schema change — back up the old file before overwriting.
+    backup_path = os.path.splitext(file_path)[0] + ".bak"
+    try:
+        xbmcvfs.copy(file_path, backup_path)
+    except Exception as e:  # pylint: disable=broad-except
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)